django/db/backends/mysql/base.py

```diff
@@ -262,13 +284,20 @@ def get_new_connection(self, conn_params):
         return conn
 
     def init_connection_state(self):
+        assignments = []
         if self.features.is_sql_auto_is_null_enabled:
+            # SQL_AUTO_IS_NULL controls whether an AUTO_INCREMENT column on
+            # a recently inserted row will return when the field is tested
+            # for NULL. Disabling this brings this aspect of MySQL in line
+            # with SQL standards.
+            assignments.append('SQL_AUTO_IS_NULL = 0')
+
+        if self.isolation_level_value:
```

Comment:
This attribute does not exist if `isolation_level` has not been specified in `OPTIONS`, AFAICT